components/butterfly/src/server/mod.rs

```diff
@@ -1141,22 +1164,6 @@ impl Server {
     pub fn is_departed(&self) -> bool { self.departed.load(Ordering::Relaxed) }
 }
 
-impl Serialize for Server {
-    fn serialize<S>(&self, serializer: S) -> result::Result<S::Ok, S::Error>
-        where S: Serializer
-    {
-        let mut strukt = serializer.serialize_struct("butterfly", 7)?;
-        strukt.serialize_field("member", &self.member_list)?;
-        strukt.serialize_field("service", &self.service_store)?;
-        strukt.serialize_field("service_config", &self.service_config_store)?;
-        strukt.serialize_field("service_file", &self.service_file_store)?;
-        strukt.serialize_field("election", &self.election_store)?;
-        strukt.serialize_field("election_update", &self.update_store)?;
-        strukt.serialize_field("departure", &self.departure_store)?;
-        strukt.end()
-    }
-}
-
```

Comment:
Why was this removed?

Reply:
It didn't appear that it was used. Since `Server` doesn't derive `Serialize` and removing this `impl Serialize for Server` didn't cause any complication errors, I concluded all serialization needs were being handled by `ServerProxy`. Removing `impl<'a> Serialize for ServerProxy<'a>` *does* break the build. Did I miss something?

Reply:
Ah, that's right.  I forgot about `ServerProxy`.